docs/doc_test/!extensions.gte

```diff
@@ -2,17 +2,17 @@ GoTemplate Extension can contains plain text at the root level.
 
 Template to define algorithm to draw a fir.
 @define("draw_fir")
-  $colors := printf("Fg%s,Bg%s", fg, bg)
-  $len := size * 2 * lenc(char) + len(color($colors))
-  @r := to(1, size*2, 2)
-  @-if (reverse) @r = r[-1:0];
-  @-foreach ($n := r)
+  @{colors := printf("Fg%s,Bg%s", fg, bg)}
```

Comment:
I thought the `{}` was always around the variable why is it sometimes like this: `@{var := "123"}` and sometimes like this: ``@{var} := "123"`